content/blog/open-source-monitoring-at-the-hpe-customer-technology-center-böblingen.md

```diff
@@ -0,0 +1,412 @@
KawmfTfcS9dYU9/cfqSTRUamCt8NFe9PhX/dpVapBnvM2K6H4eXAs2n0QQrElwNZD3cdvsMDNEgWrZZ0I9v9HvzvYFLUzdQTB+NmHi/M7WTgqE969NJ1JHkbjz19LhZH/Z25nahPc2zZvIfeZ2pFG/9JThFnJ5xL25hK3N4yYcdWdmr60czWeyxeLv8jMVw6kscxbD7Kmmy6kJFQpNNcI/fEpvuT96bnkqfbV914b3u6jkXrCaP0aazMza0k0NSYBvu/Vsz6jbVa4/VVOA0P6jbt27BWeWYD3rNa8z03R7GmTQdpP1zl2tjaP9/gtvSQrkZBAek0+8tq9mJWQ0rg4WvOS7qzGmUyeHzXcaGij21thy+Z6T/d/i1i0nrmkokDpJMHzOo9fXLfwAW+f7t0ZW2laDu67RR95Nxnt4lpXhrpuWzTTP5kC0XSjq5aGnqThHbzFeoufvfcesz8eefkPu+ZmhLSYbCj9MqUJgOmO06f2idw/va/P2JNNXmguqftMHvuPa9Vi47p6hKJRO8tlyU00xq/Nybo+0Vuh3RpsXN7lt/MJ8wb5PX9sq/8aKvpmP7cQcWUpsMsl3s/rVx0nL+/rqPLl2Xjnq6VrfHBDYsu0BkKCoz7fbag7AxmHd/NDYh9PrAt9xWN74z0KdMZmR0bVjh4Cd2zdSfKd8ECvqCmafbW9HkDuCjIb9r3i87p6hZL6OYvHMsNuJqPnTfE+3u3pdxuMBvdX673VdPBof3ePWREj3ruF+PQtRq8acPXSwn9Q5MUW0xYYFPVSih+4hxm/V/sOmGT9eztbAn/1mbcd0zvS75uS/Zq6tqN6MG6pxU+Yvvuxv6/LgqniywsMO772QTu+5Gx46D2nj6LovvOc5/IHz/Ds530mT29+xLuLD/JC4l5/4WfcX+VwlpJ93kH5yUOlW2HhdoNT7dDNL/y68/PauI4pJ3nhq+i+811n9BZmKtucH8OGzYs/YroahZIiMXYhTb0K6Sil7ptv+H+v7gtipmw6rN+tnbpG5Z9dYjepZB0mrFUde+GFe1PRX/dpVfJUH6PCeNxFWk1/PO5QTt/WxmYp2fxzmB7E/6rcN29wwM0UHoDF7Q7v/vR3FMZm4K4b2JT3zH2mcmfdtPVcvcHcXP9Hhsfp29m6os/aP0RF5QsflyQP2Nzetur6dzM/U2/fZu+Wci+yBlNnSs+/p34f38k//uJ7HIHBiMXmP7qmerwzS1asTLS3TqXLl992qTMkN8zjOdz5+kM7Grw68iyH6a1UdPrkFesuECSXcVFcwuyxYXaNbysbp5EXFwvl+JVhbwMMXe9cuXGmnPEYnXpxc6rUqsHUrXyF4vmLlasZsj1T8ur6KrKeRIJ0S07sxzFS2PS9/l46Mz6aWIF/XjPw7+cnr3o+NAy53TXE4VrrnA3FFze/PV1R+/yI/v1pqJnpxKKnjj2YrjuuyDCQcGBCvIUPCLfpK/gwuMK0HeGPKJbbmaF+7yM6G3c2rl0L5DQJbyy95YKtr+Sla/Ge6zyKtqfFVwKvqYKCgo0NYUVTzm2ylfrU340n2pU7/AAKlKYKy7QMdRltRIK24tys/OInp5Ojf4M83KeE+3mZVJCTnaupr5O7f7ayl+HvNZvDzJqmrpVvRlo6tf87UK7yqU2INpGtch2etUNkVStHkjVNMt99nAvhfLbUtELRLuyEEkpXhpjPG2kXUj4bVYro/D2wVtd1vZ/NSGSUrjm5XZDUvQJ/82HUvoNfnUhkqro2amEoieu/IuhQgoekW+q5t3pO4OimRXuc8XoW9arfG+pYPsrWflqvMcqr6L9Wf0ntBLpZzcsXbszRHTntih4j2+Idm872S9xNKp3eAAV0VAUIimF7eo6+jUMkZS2XtkQSenVNkQqpLr+SABQQl6K6HqSZjs7m2r9uF8j8Twh4rmxg2W9H+1ZPeL7EenN2VUnoZ6IkyIuhN1Iy2lm7tSvr43CE74AoOEr3x+JHAkAAAAAVavLcW0AAAAA+C9BjgQAAAAAZSBHAgAAAIAykCMBAAAAQBnIkQAAAACgDORIAAAAAFDGf/K6P/97mxVeY79eZAUAAAAAVcB1fwAAAABANZqkp6ezIgAAAABAxfB7NvD6KN/BDgAAAHUE49oAAAAAoBrIkQAAAACgDIxrQyOGcW0AAIC6EBERcf/+fVqwtLR0cHAQGjGuDQAAAACVoSHy7t27BTxaoFU2oRzkSAAAAAAoIfREypSpykOOBAAAAIASBQUFrMQrU5WHHAkAAAAAykCOBAAAAABlIEcCAAAAgDKQIwEAAABAGciRAAAAAKAM5EgAAAAAUAZyJAAAAAAoAzkSAAAAAJSBHAkAAAAAykCOBAAAAABlIEcCAAAAgDKQIwEAAABAGciRAAAAAKCMJi9fvmRFqEdFRUX5+fmFhYWsDlIaGhpaWlrq6uqsXqlnz561aNGCVQAAAEAV9u/fz0pSU6dOpf+W/9hFf+QrQENkTk4OQqRCdLfQnUN3EasDAABA/dLU1GQlXpmqPOTIVyA/P5+VoALYRQAAAK+KpaUlK/HKVOUhR74C6ImsEnYRAADAq+Lg4NCxY0dNHi3QKptQDo6PfAWysrJYCSpmYGDAShXD8ZEAAAD1BsdHAgAAAIBqIEcCAAAAgDKqHtdOP7PB61QKq0iZDV2ysL8xq0ANYVy7OjCuDQAA0KAoM66t2cLKhmNhmCcW5xla8BWrFhWeAQ4AAAAA/wXVP88mJWjN2mNkzLLlw81oLSl8b0isxYDZ/d7gpiWc2XkmtduY4bphh6806zfR7F5gWBKxeHvEYBtDbjIhBWnRISHRj/OadRow2PEN1vifhf7I6kB/JAAAQIOiuvNsTDTFERGBEQlcuTg6+GCEqNDQWJIUczkieMcGv9s5kgfh/hvW+MZI6PSC6zvdVvmeSSokefeO/eTmebLsKDkAAAAANDrK5khtB0c7Io6JTqLl26LYYkMnRxthCrGbvuyz2fO+XTjYWBJ94mw6EYf/
```

Comment:
when using "underscore", make sure to escape the "underscore" because it is a special markdown character:

The way to escape a special character is to add a backslash (\\) before it, for example: I do not want \_italic text\_ here.

Please refer to tip 13 in the CMS editor guide.